### PR TITLE
LPD-20898 - Remove FF LPS-186870 CK Editor from Release Feature Flags

### PR DESCRIPTION
**Pull request by kresimir-coko**
Quick and dirty, unsure if this is the correct approach as the ticket is scarce of information outside of the title.

### CODE REVIEW

File: modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts [hunk 1]
```diff
@@ -24,13 +24,12 @@ export const test = mergeTests(
...
 	featureFlagsTest({
 		'LPS-178052': true,
-		'LPS-186870': true,
 	}),
 	isolatedSiteTest,
 	loginTest()
 );
 
-test('Create, edit and delete editor config contributor client extension @LPS-186870', async ({
+test('Create, edit and delete editor config contributor client extension', async ({
```
Comment (markocikos) [inline modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts:32]: You can leave this in. It's a tag, that matches epic ticket. It just servers for some convenient filtering and searching. [Suggested here](https://liferay.slack.com/archives/C06A59K5D46/p1707314754276979?thread_ts=1707259884.390479&cid=C06A59K5D46).

**Review by markocikos — CHANGES_REQUESTED** [review]
@kresimir-coko A minor bit inline. But, this PR does not cover it all. Searching DXP for `LPS-186870` I also find it in portal properties and lang keys

**Comment by kresimir-coko** [comment]
Seems the `portal.properties` change was left uncommited, but great tip about the language keys.
Fixed all 3 things and force pushed

- [ ] Remove mentions from `portal.properties`
- [ ] Remove mentions from `Language.properties`
- [ ] Discard the annotation mentions

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
@kresimir-coko LGTM! :sparkles:

**Comment by kresimir-coko** [comment]
ci:report:2561923433

**Comment by kresimir-coko** [comment]
Testray report shows no unique failures

**Comment by kresimir-coko** [comment]
ci:forward

**Comment by kresimir-coko** [comment]
ci:forward

**Comment by kresimir-coko** [comment]
ci:forward

**Comment by kresimir-coko** [comment]
ci:forward

**Comment by markocikos** [comment]
@kresimir-coko Don't just `ci:forward` forever. You can analyse if the failures are related, and force forward or manual forward if they are not.

**Comment by kresimir-coko** [comment]
Manually forwarded to https://github.com/brianchandotcom/liferay-portal/pull/149621

**Pull request closed without merging**